Source/Scene/Cesium3DTileset.js

```diff
@@ -264,6 +264,7 @@ define([
 
         this._maximumScreenSpaceError = defaultValue(options.maximumScreenSpaceError, 16);
         this._maximumNumberOfLoadedTiles = defaultValue(options.maximumNumberOfLoadedTiles, 256);
+        this._maximumMemoryUsage = defaultValue(options.maximumMemoryUsage, 256);
```

Comment:
Obviously this is dependent on hardware, but I think 256 is a bit low as a default. 512 seems to be a good amount at least on my machine.